cardano-wallet.cabal

```diff
@@ -0,0 +1,60 @@
+name:                cardano-wallet
```

Comment:
This file shouldn't exist. We moved executables into their corresponding packages.